2_base_and_sensor.ipynb

```diff
@@ -172,40 +316,78 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "壁にぶつかってしまいました。上の■ボタンを押して動作を停止させましょう。\n",
     "![title](./imgs/2_stop_button.png)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Oops, the robot bumped into the wall. Press the ■ button above to stop our code.\n",
+    "![title](./imgs/2_stop_button.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "一定時間動かし続けてみましょう。以下を実行すると3秒間直進させることができます。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's keep it running for a limited period of time. Following code will let the robot go straight for 3 seconds:"
```

Comment:
"Let's keep it running for a limited period of time. The following code will make the robot go forward for 3 seconds:"